engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/job/preparation/JobPreparationActor.scala

```diff
@@ -187,7 +187,10 @@ class JobPreparationActor(workflowDescriptor: EngineWorkflowDescriptor,
                                                     runtimeAttributes: Map[LocallyQualifiedName, WomValue],
                                                     maybeCallCachingEligible: MaybeCallCachingEligible,
                                                     prefetchedJobStoreEntries: Map[String, KvResponse]): BackendJobPreparationSucceeded = {
-    val jobDescriptor = BackendJobDescriptor(workflowDescriptor.backendDescriptor, jobKey, runtimeAttributes, inputEvaluation, maybeCallCachingEligible, prefetchedJobStoreEntries)
+    val backendWorkflowDescriptor = workflowDescriptor.backendDescriptor
+    val jobDescriptor = BackendJobDescriptor(backendWorkflowDescriptor, jobKey, runtimeAttributes, inputEvaluation,
```

Comment:
ToL: IMO if you're going to separate lines, separate everything onto their own lines...